src/main/java/org/activiti/cloud/qa/service/RuntimeBundleService.java

```diff
@@ -46,4 +46,13 @@ void assignTaskToUser(@Param("id") String id,
 
     @RequestLine("POST /v1/tasks/{id}/complete")
     void completeTask(@Param("id") String id);
+
+    @RequestLine("DELETE /v1/process-instances/{id}")
+    @Headers("Content-Type: application/json")
+    void deleteProcess(@Param("id") String id);
```

Comment:
@constantin-ciobotaru quick question here.. why do you call it delete if it is cancelled? (is that because of the HTTP METHOD?) maybe the name of the java method should be cancelProcessInstance, it should be more clear.

Reply:
Because, from the engine point of view, the process instance is deleted. The engine's method name is org.activiti.engine.RuntimeService.deleteProcessInstance(). But, sure, this is the client point of view, which sees it as a cancel. I'll update.

Reply:
if the engine method is delete, then that's fine.. i didn't knew that we were doing that..